src/main/java/component/Header.java

```diff
@@ -0,0 +1,19 @@
+package component;
```

Comment:
this `component` package contains elements of the page so move it to the test directory